### PR TITLE
Release notes for 2.5.0

### PR DESCRIPTION
**Pull request by gassmoeller**
A suggestion for release notes for the 2.5.0 release.

Related to #5145.

### CODE REVIEW

File: contrib/release/release-tasklist.md [hunk 1]
```diff
@@ -192,6 +192,96 @@ and the links are working
...
+
+        https://github.com/geodynamics/aspect/releases/tag/v2.5.0
+
+Among others this release includes the following significant changes:
+
+- ASPECT now includes version 0.5.0 of the Geodynamic World Builder.
+  (Menno Fraters and other contributors)
+
+- ASPECT's manual has been converted from LaTeX to Markdown to be hosted as a
+  website on https://aspect-documentation.readthedocs.io.
```
Comment (tjhei) [inline contrib/release/release-tasklist.md:221]:
```suggestion
  website at https://aspect-documentation.readthedocs.io.
```
?

File: contrib/release/release-tasklist.md [hunk 2]
```diff
@@ -192,6 +192,96 @@ and the links are working
...
+
+- Changed: ASPECT now releases the memory used for storing initial conditions
+  and the Geodynamic World Builder after model initialization unless an
+  owning pointer to these objects is kept. This reduces the memory footprint
+  for models initialized from large data files.
+  (Wolfgang Bangerth)
+
+- Added: Various helper functions to distinguish phase transitions for
+  different compositions and compositional fields of different types.
+  (Bob Myhill, 2023/01/19)
```
Comment (tjhei) [inline contrib/release/release-tasklist.md:247]: You deleted dates for other entries but not here it seems.

File: contrib/release/release-tasklist.md [hunk 3]
```diff
@@ -192,6 +192,96 @@ and the links are working
...
+
+- New: ASPECT now requires deal.II 9.4 or newer.
+  (Rene Gassmoeller, Timo Heister)
+
+- ASPECT now supports a DebugRelease build type that creates a debug build and
+  a release build of ASPECT at the same time.  It can be enabled by setting the
+  CMake option CMAKE_BUILD_TYPE to DebugRelease or by typing "make debugrelease".
+  (Timo Heister)
+
+- ASPECT now has a CMake option ASPECT_INSTALL_EXAMPLES that allows to build
```
Comment (bangerth) [inline contrib/release/release-tasklist.md:233]: The correct use of the phrase is "allows <noun>" where the noun can be "doing". 
```suggestion
- ASPECT now has a CMake option ASPECT_INSTALL_EXAMPLES that allows building
```

File: contrib/release/release-tasklist.md (hunk 2 above)
Comment (bangerth) [inline contrib/release/release-tasklist.md:247]:
```suggestion
  (Bob Myhill)
```

File: contrib/release/release-tasklist.md [hunk 4]
```diff
@@ -192,6 +192,96 @@ and the links are working
...
+  half-space cooling model.
+  (Daniel Douglas, John Naliboff, Juliane Dannberg, Rene Gassmoeller)
+
+- New: ASPECT now supports tangential velocity boundary conditions with GMG for
+  more geometries, such as 2D and 3D chunks.
+  (Timo Heister, Haoyuan Li, Jiaqi Zhang)
+
+- New: Phase transitions can now be deactivated outside a given temperature
+  range specified by upper and lower temperature limits for each phase
+  transition. This allows to implement complex phase diagrams with transitions
```
Comment (bangerth) [inline contrib/release/release-tasklist.md:262]:
```suggestion
  transition. This allows implementing complex phase diagrams with transitions
```

**Review by bangerth — COMMENTED** [review]
OK with the minor edits.

**Comment by gassmoeller** [comment]
Thanks for the suggestions, should be fixed.

**Comment by bangerth** [comment]
Please squash, if you wouldn't mind.

**Comment by gassmoeller** [comment]
No problem, done.

**Review by bangerth — APPROVED** [review]

**Pull request merged**